### ISSUE TITLE
Migration to Flutter 2 and Null safety

### ISSUE THREAD

**Issue by sergey-regotun** [roughike/flutter_facebook_login#321]
@roughike thank you for the plugin!

Is this plugin going to migrate to Flutter 2 and Null safety?

**Comment by daniellampl** [comment]
I don't think that this package will get an update any time soon. But there is a great alternative: [flutter_facebook_auth](https://pub.dev/packages/flutter_facebook_auth)
